### DISCUSSION TITLE
Is issue.user.login the actual issue creator?

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#26029] [Actions]
<p>I currently have a auto-commenter action setup that responds to common issues based on assigned labels.<br>
This action right now has a <code>{{ sender.login }}</code> placeholder to get replaced with a name.</p>
<p>Unfortunately as I recently found out, is <code>sender.login</code> actually whoever caused the issue action (Makes sense).<br>
I however want the actual issue creator in order to respond with the right name.<br>
Looking at the <a href="https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads#pull_request" rel="nofollow noopener">Webhook events and payloads page</a> does it seem like the actual part I want to use is <code>issues.user.login</code>.</p>
<p>Is this the actual user who created (opened) the issue, or is there a possibility that this object would also change over time?</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/Andre601">@Andre601</a>,</p>
<p>Yes, you can get the creater from the <code>user</code> object in webhook payload, it’s not changed.</p>
<p>Your link points to pull request payload, the creater is <code>${{ github.event.pull_request.user.login }}</code>.</p>
<p>For common issue, the creater is <code>${{ github.event.issue.user.login }}</code>.</p>
<p>Thanks</p>
